source/_UIAHandler.py

```diff
@@ -140,7 +142,7 @@
 
 UIAEventIdsToNVDAEventNames={
 	UIA_LiveRegionChangedEventId:"liveRegionChange",
-	#UIA_Text_TextChangedEventId:"textChanged",
+	UIA_Text_TextChangedEventId:"textChange",

```

Comment:
Having thought some more about this... Note that there was a good reason in the past to comment this out, see https://github.com/nvaccess/nvda/commit/9025c04dc0aaef7fc5ae4575844260503b5acdc5 and issue #1205.

We should consider this carefully.

Reply:
At very least, we should only register for this event on Windows 10 and above. I don't think we can  hold back because of .net 3.5 for ever, as these are very useful events. Plus, Windows 10 now has improved UIA event filtering which should hopefully  solve the issue on windows 10.